airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/manifest.yaml

```diff
@@ -150,11 +161,137 @@ definitions:
         field_name: as_of_time
         inject_into: request_parameter
     $parameters:
-      path: "balances"
+      path: "v1/reporting/balances"
+
+  #Stream search Invoices 
```

Comment:
I see this has `invoice_date_range` in the filter options (in the body schema?)
And the response has `metadata.create_time` --- I suppose these are mutable and that doesn't work, huh?

Reply:
So we could use any of the X_date_range. The reason of originally using created_date_range was the ability to section the time by Hrs and minutes, whereas invoice_date_range is per day. That was originally when I was testing for incremental sync expecting the update_time as per the output definition. We could think on providing both options, especially if willing just to get all invoices per day, rather than by chunks of time.  
I'll look into it (not urgent)